examples/angular/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "angular",
```

Comment:
To align with the other examples, could we name this `near-wallet-selector-example-angular`

Reply:
Thanks, did not notice it :)